test/cppsuite/tests/test_live_restore.cpp

```diff
@@ -311,72 +306,138 @@ get_stat(WT_CURSOR *cursor, int stat_field, int64_t *valuep)
+ *  default is off.
+ *  -l log level, this controls the level of logging that this test will run with. This is distinct
+ *  from verbose level as that is a WiredTiger configuration. Default is LOG_ERROR (0). The other
+ *  levels are LOG_WARN (1), LOG_INFO (2) and LOG_TRACE (3).
+ */
+#include <iostream>
+int
+main(int argc, char *argv[])
+{
+    /* Set the program name for error messages. */
+    const std::string progname = testutil_set_progname(argv);
+
+    // Parse the options.
+    auto log_level_str = value_for_opt("-l", argc, argv);
+
+    // Set the tracing level for the logger component.
+    logger::trace_level = log_level_str.empty() ? LOG_ERROR : atoi(log_level_str.c_str());
+
+    // Get the iteration count if it exists.
+    auto it_count_str = value_for_opt("-i", argc, argv);
+    int64_t it_count = it_count_str.empty() ? iteration_count_default : atoi(it_count_str.c_str());
+    logger::log_msg(LOG_INFO, "Iteration count: " + std::to_string(it_count));
+
+    // Get the thread count if it exists.
+    auto thread_count_str = value_for_opt("-t", argc, argv);
+    int64_t thread_count =
+      thread_count_str.empty() ? thread_count_default : atoi(thread_count_str.c_str());
+    logger::log_msg(LOG_INFO, "Background thread count: " + std::to_string(thread_count));
+
+    // Get the op_count if it exists.
+    auto coll_count_str = value_for_opt("-c", argc, argv);
+    int64_t coll_count = INT64_MAX;
+    if (!coll_count_str.empty()) {
+        coll_count = atoi(coll_count_str.c_str());
+        logger::log_msg(LOG_INFO, "Collection count: " + std::to_string(coll_count));
+    }
+
+    // Get the op_count if it exists.
+    auto op_count_str = value_for_opt("-o", argc, argv);
+    int64_t op_count = op_count_str.empty() ? op_count_default : atoi(op_count_str.c_str());
+    logger::log_msg(LOG_INFO, "Op count: " + std::to_string(op_count));
+
+    // Get the verbose_level if it exists.
+    auto verbose_str = value_for_opt("-v", argc, argv);
+    int64_t verbose_level = verbose_str.empty() ? 0 : atoi(verbose_str.c_str());
+    logger::log_msg(LOG_INFO, "Verbose level: " + std::to_string(verbose_level));
+
+    // Background thread debug mode option.
+    bool background_thread_debug_mode = option_exists("-b", argc, argv);
+
+    // Home path option.
+    std::string home_path = value_for_opt("-h", argc, argv);
+    if (home_path.empty())
+        home_path = HOME_PATH;
+    logger::log_msg(LOG_INFO, "Home path: " + home_path);
+
+    // Delete any existing source dir.
+    testutil_recreate_dir(SOURCE_PATH);
+    logger::log_msg(LOG_INFO, "Source path: " + std::string(SOURCE_PATH));
+
+    // Recreate the home directory on startup every time.
+    testutil_recreate_dir(home_path.c_str());
+
+    bool first = true;
+    /* When setting up the database we don't want to wait for the background threads to complete. */
+    bool background_thread_mode_enabled = (!first && background_thread_debug_mode);
```

Comment:
This doesn't seem like a `debug` mode to me anymore, just a background thread mode. This applies to the help comment too.

Reply:
Well it basically turns off half the test program to help debug background thread issues. Is that not what a debug mode is?

Reply:
Right, because the concurrency isn't supported yet, but after that we can run them together